src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -415,6 +412,26 @@ void cannotCreateTitleLevelRequestForUnknownInstance(String tlrFeatureStatus,
     assertThat(postResponse.getJson(), hasErrorWith(hasMessage("Instance does not exist")));
   }
 
+  @ParameterizedTest
+  @CsvSource({"Page", "Hold", "Recall"})
+  void cannotCreateTitleLevelRequestForUnknownInstance(String requestType) {
```

Comment:
Why was the original test split? Also, the original test covered different statuses of TLR feature - NOT_CONFIGURED, DISABLED, ENABLED

Reply:
but they were only for Item level requests and for TLR only used ENABLED